openlibrary/templates/authors/infobox.html

```diff
@@ -0,0 +1,33 @@
+$def with (page, edit_view: bool = False)
+
+$ is_librarian = ctx.user and (ctx.user.is_librarian() or ctx.user.is_super_librarian() or ctx.user.is_admin())
+
+$if edit_view:
+    $ wikidata = page.wikidata(bust_cache=True, fetch_missing=True)
+$else:
+    $ wikidata = page.wikidata(fetch_missing=is_librarian)
+
+$def render_infobox_row(label, itemprop, value):
+    $if value:
+        <tr>
+            <td><strong>$_(label)</strong></td>
+            <td><span itemprop="$itemprop">$value</span></td>
+        </tr>
+
+<div class="infobox">
+    <div class="illustration">
+        $:render_template("covers/author_photo", page)
+        $:render_template("covers/change", page, ".bookCover img")
+    </div>
+    <p class="short-description">
+        $if wikidata:
+            $wikidata.get_description(i18n.get_locale())
+    </p>
+    <table>
+        $if page.birth_date or page.death_date:
+            $:render_infobox_row("Born", 'birthDate', page.birth_date)
+            $:render_infobox_row("Died", 'deathDate', page.death_date)
```

Comment:
Little gotcha on the i18n here; the _ function is analyzed statically before the code is executed, so it only supports string literals.

```suggestion
            $:render_infobox_row(_("Born"), 'birthDate', page.birth_date)
            $:render_infobox_row(_("Died"), 'deathDate', page.death_date)
```